.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: "\U0001F41B Bug report"
+description: Create a report for a bug you have found, or regression you have noticed
+title: 'BUG: '
+labels: [bug]
+assignees: ''
+
+---
+
+**Describe the bug**
+A short description of what the bug is.
+
+**Steps to Reproduce**
+Steps to reproduce the behavior:
+1. Code/test/example that was run  '...' (as minimal as possible)
+2. Command executed '...'
+3. See error
```

Comment:
How is this different to "Error message" below?

Reply:
See error here is just the last step. I will remove if it's confusing